samples/misc/vaultGetPoolTokens.py

```diff
@@ -1,23 +0,0 @@
-import balpy
```

Comment:
any reason to remove this one?

Reply:
Because I added this on the `vaultGeneralRead` sample. Not sure if we should do this way but I think that is better than having one file for each function.